app/mailers/late_reservation_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class LateReservationMailer < ApplicationMailer
+	default from: 'notifications@example.com'
+
+	def forgot_to_buy_gift_email(user)
```

Comment:
Tabs don't have a fixed length - their length is determined by the settings of the IDE of the person viewing the file.

You can set Sublime settings on a per-language basis, and I'd recommend setting it to auto-convert tabs to spaces on a 1 tab -> 2 spaces ratio for Ruby.